packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -84,6 +90,7 @@ exports.config = {
     // https://docs.saucelabs.com/reference/platforms-configurator
     //
     capabilities: [{
+    <%if(answers.exp_env_access_key){%> accessKey: '<%= answers.exp_env_access_key %>',<%}%>
```

Comment:
this should be added by WebdriverIO as it does already with Sauce Labs or BrowserStack

Reply:
this is not W3C conform .. is Experitest planning to use vendor capabilities?

Reply:
This 'accessKey' is Experitest capability and used for authentication in Experitest Cloud.
please let me know if it's ok so I can push all your suggestions

Reply:
@ArielExperitest .. I am assuming you work for Experitest. This key is invalid. According to the spec the following capabilities are allowed: https://w3c.github.io/webdriver/#capabilities. You have to vendor prefix it like `"experitest:accessKey"`.